client/src/entrypoints/snippets/snippet-chooser.js

```diff
@@ -42,9 +44,18 @@ function createSnippetChooser(id, modelString) {
       state = newState;
     },
     openChooserModal: () => {
+      let urlQuery = '';
+      if (wagtailConfig.ACTIVE_CONTENT_LOCALE) {
```

Comment:
This isn't currently being defined anywhere.

Reply:
Added to page/snippet create/edit views, thanks!